script.js

```diff
@@ -38,3 +38,27 @@ document.addEventListener("DOMContentLoaded", function () {
     closeButton.style.display = "none";
   }
 });
+
+const form = document.querySelector("form");
+const errorMessage = document.createElement("p");
+errorMessage.style.color = "red";
+
+function validateEmail() {
+  const emailInput = document.querySelector('input[type="email"]');
+  const email = emailInput.value;
+
+  if (email !== email.toLowerCase()) {
+    errorMessage.textContent = "Please enter the email address in lowercase.";
+    form.appendChild(errorMessage);
+    return false;
+  }
+
```

Comment:
- [ OPTIONAL]  I think it would be better if the error message appears right below the email field instead of at the bottom of the form. What do you think 👍🏾